randovania/games/prime1/exporter/patch_data_factory.py

```diff
@@ -701,7 +701,10 @@ def create_data(self) -> dict:
                         self.rng,
                     )
 
-                    if node.extra.get("position_required"):
+                    if self.configuration.shuffle_item_pos:
+                        aabb = area.extra["aabb"]
+                        pickup["position"] = _pick_random_point_in_aabb(self.rng, aabb, area.name)
+                    elif node.extra.get("position_required"):
```

Comment:
Wouldn't it be better to combine these two if blocks since two lines are identical?